server/routerlicious/packages/services-shared/src/storage.ts

```diff
@@ -197,8 +198,10 @@ export class DocumentStorage implements IDocumentStorage {
             isSessionAlive: true,
```

Comment:
In alfred/routes/api/documents.ts (where this function is called), a similar session object is created and returned to the user. However, that ISesssion says `isSessionAlive: false`, and this one says `isSessionAlive: true`. Are they supposed to be different?

Reply:
Yup. :) They should be different, the clients should get the session status before they open this session. :)